routes/api/v1/meals.js

```diff
@@ -82,4 +82,33 @@ router.post('/:mealId/foods/:foodId', function(req, res) {
   });
 });
 
+router.delete('/:mealId/foods/:foodId', function(req, res) {
+  Meal.findByPk(req.params.mealId)
+  .then(meals => {
+    if (meals) {
+      return Food.findByPk(req.params.foodId)
+      .then(foods => {
+        if (foods) {
+          return meals.removeFood(foods)
```

Comment:
It's funny (not funny 'haha' but funny 'ugh) that no where in the docs could we find this stupid one-word change from the addFood method. Wishing those docs were better, and very glad we have informed colleagues who know what's up.